Failed to compile Jannovar v0.20

Compilation (e.g. `mvn package`) failed for Jannovar v0.20 for fresh checkout.

Checkout
```
$ git clone https://github.com/charite/jannovar.git jannovar
$ cd jannovar
```

switch to v0.20
```
$ git checkout v0.20
Note: checking out 'v0.20'.
```

Environment:

Ubuntu 16.04.1 LTS - 64bit
```
$ java -version
java version "1.8.0_111"
```

Error:
```
...
Tests run: 4, Failures: 2, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Jannovar ........................................... SUCCESS [  0.850 s]
[INFO] de.charite.compbio:jannovar-hgvs ................... SUCCESS [  8.310 s]
[INFO] de.charite.compbio:jannovar-core ................... SUCCESS [  9.394 s]
[INFO] de.charite.compbio:jannovar-inheritance-checker .... SUCCESS [  2.556 s]
[INFO] de.charite.compbio:jannovar-htsjdk ................. SUCCESS [  4.512 s]
[INFO] de.charite.compbio:jannovar-vardbs ................. SUCCESS [  3.619 s]
[INFO] de.charite.compbio:jannovar-cli .................... FAILURE [  2.504 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 31.927 s
[INFO] Finished at: 2017-01-09T15:59:44+01:00
[INFO] Final Memory: 53M/817M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project jannovar-cli: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/mjaeger/git/jannovarTEST/jannovar-cli/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :jannovar-cli
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Jannovar Changelog
 
+### all
+
+* Fixing language in mvn surfire plugin. Now mvn tests work on locale de_DE etc..
+
 ## develop
 
 ### jannovar-htsjdk

--- a/pom.xml
+++ b/pom.xml
@@ -114,10 +114,10 @@
                     <skipTests>false</skipTests>
                     <forkCount>3</forkCount>
                     <reuseForks>true</reuseForks>
-                    <argLine>${argLine} -Xms512m -Xmx1024m</argLine>
+                    <argLine>${argLine} -Xms512m -Xmx1024m -Duser.language=en -Duser.region=US</argLine>
                     <systemPropertyVariables>
                         <user.language>en</user.language>
-                        <user.region>GB</user.region>
+                        <user.region>US</user.region>
                     </systemPropertyVariables>
                 </configuration>
             </plugin>

